e2e/protractor/suites/actions-available/special-permissions/other-permissions.ts

```diff
@@ -323,49 +201,5 @@ export function filesLockedByOtherUser(siteName?: string) {
 
       await testUtil.checkToolbarActions(testData.fileLockedByUser, expectedToolbarPrimary, expectedToolbarMore);
     });
-
-    describe('available actions in the viewer : ', () => {
-      it('file opened from File Libraries - [C297671]', async () => {
-        await page.goToMyLibrariesAndWait();
-        await dataTable.doubleClickOnRowByName(siteName);
-        await dataTable.waitForHeader();
-
-        const expectedToolbarPrimary = ['Activate full-screen mode', 'Shared Link Settings', 'Download', 'Print', 'View Details', 'More Actions'];
-        const expectedToolbarMore = ['Cancel Editing', 'Remove Favorite', 'Move', 'Copy', 'Delete', 'Manage Versions', 'Permissions'];
-
-        await testUtil.checkViewerActions(testData.fileLockedByUser, expectedToolbarPrimary, expectedToolbarMore);
-      });
-
-      it('file opened from Shared Files - [C297672]', async () => {
-        await page.clickSharedFilesAndWait();
-
-        const expectedToolbarPrimary = ['Activate full-screen mode', 'Shared Link Settings', 'Download', 'Print', 'View Details', 'More Actions'];
-        const expectedToolbarMore = ['Cancel Editing', 'Remove Favorite', 'Move', 'Copy', 'Delete', 'Manage Versions', 'Permissions'];
-
-        await testUtil.checkViewerActions(testData.fileLockedByUser, expectedToolbarPrimary, expectedToolbarMore);
-      });
-
-      it('file opened from Favorites - [C297673]', async () => {
-        await page.clickFavoritesAndWait();
-
-        const expectedToolbarPrimary = ['Activate full-screen mode', 'Shared Link Settings', 'Download', 'Print', 'View Details', 'More Actions'];
-        const expectedToolbarMore = ['Cancel Editing', 'Remove Favorite', 'Move', 'Copy', 'Delete', 'Manage Versions', 'Permissions'];
-
-        await testUtil.checkViewerActions(testData.fileLockedByUser, expectedToolbarPrimary, expectedToolbarMore);
-      });
-
-      it('file opened from Search Results - [C306994]', async () => {
-        await toolbar.clickSearchIconButton();
-        await searchInput.clickSearchButton();
-        await searchInput.searchFor(testData.fileLockedByUser);
-        await searchResultsPage.waitForResults();
-
-        const expectedToolbarPrimary = ['Activate full-screen mode', 'Shared Link Settings', 'Download', 'Print', 'View Details', 'More Actions'];
-        // TODO: add 'Move' and 'Delete' when ACA-2319 is fixed
```

Comment:
Should we add this comment to new playwright test? I can see that new playwright test have not Move and Delete: 
<img width="1244" alt="image" src="https://github.com/Alfresco/alfresco-content-app/assets/115619721/b4838803-450a-4b2d-965a-0aba1595fbb8">


If we don't add this comment we can forget to add these two options when this jira will be fixed.

Reply:
Actually this Jira was fix in past and comment is from old protractor test, I ask Agnieszka and raise a separate ticket for this issue.(https://alfresco.atlassian.net/browse/ACS-6260) So I removed this commented line.

Reply:
@akashrathod28 fine but in that case if old jira was fix then why it is still in open status? Should not this be closed jira?
<img width="1728" alt="image" src="https://github.com/Alfresco/alfresco-content-app/assets/115619721/72c66af8-e9b7-4bff-acc6-2c289b515e22">

As you can see ACA-2319 is still open.

Reply:
Yes jira was mark done but the option are still visible on aca,
as this comment- they moved this ticket for future reference, So I guess we can remove this and when ever this change come we can remove this option from test(test will fail positive) 
https://alfresco.atlassian.net/browse/ACA-1737?focusedCommentId=1150288

Reply:
@akashrathod28 in comment in code there was reference to ACA-2319 and that jira is open (not closed). If you open this jira you will see. 
You reference to ACA-1737, but I'm saying about ACA-2319 which is in comment in code. 

What we should do with ACA-2319? They are two different jiras